web/src/views/InvestigationDetail.vue

```diff
@@ -498,17 +505,84 @@ export default defineComponent({
       }));
     }
 
+    async function toggleNonTiledImageOverlay(pin: Pin, dataset: Dataset) {
+      if (!dataset.content) return;
+      if (!featureLayer) return; // sanity check. this should exist
+      if (!nonTiledOverlayFeature) {
+        nonTiledOverlayFeature = featureLayer.createFeature('quad');
+      }
+      const quadData = nonTiledOverlayFeature.data() || [];
+      // eslint-disable-next-line
+      const existingOverlay = quadData.find((overlay: any) => overlay.pinId === pin.id);
+      if (existingOverlay) {
+        nonTiledOverlayFeature.data(
+          // eslint-disable-next-line
+          nonTiledOverlayFeature.data().filter((overlay: any) => overlay.pinId !== pin.id),
+        ).draw();
+        nonTiledOverlayFeature.draw();
+      } else {
+        try {
+          const urlRoot = process.env.VUE_APP_API_ROOT;
+          const url = `${urlRoot}/datasets/tile_source/${dataset.id}/thumbnail.png`;
+          const image: HTMLImageElement = await getNonTiledImage(url);
+          image.crossOrigin = 'use-credentials';
+          if (store.state.axiosInstance) {
+            const imageMetadata = await (await store.state.axiosInstance.get(`${urlRoot}/datasets/tile_source/${dataset.id}/metadata`)).data;
+            const ul = postGisToPoint(pin.child_location) || { x: 0, y: 0 };
+            const lr = {
+              x: ul.x + (imageMetadata.sizeX || 0),
+              y: ul.y + (imageMetadata.sizeY || 0),
```

Comment:
Since you already have these values from the `Image` object that contains the actual image data, I wonder if we can avoid the extra call to the metadata endpoint here (this is if the Chrome/Brave issue was solved by the type narrowing, rather than the use of the `Image` object).

Reply:
Something I noticed when testing this was that the issue with chrome seems to be that the height and width of the image element are not available right away. I'm at a loss as to why though... I managed to figure out that `image.loading` in Firefox is `eager`, and in Chrome it is `auto`. Manually setting `image.loading = 'eager'` did not seem to make a difference.

Essentially in Chrome, the height and width are not available at the time of calculating `ul` and `lr`. There was no problem loading the image (we could see that via the network tab), but the quad feature had a width and height of 0.

In 2ec5d2c I've removed the utility function, in favor of using the `/metadata` endpoint to get the image dimensions.